runtime/runtime-params-estimator/src/estimator_context.rs

```diff
@@ -65,23 +65,34 @@ impl<'c> Testbed<'c> {
         &mut self.transaction_builder
     }
 
+    /// Apply and measure provided blocks one-by-one.
+    /// Because some transactions can span multiple blocks, each input block
+    /// might trigger multiple blocks in execution. The returned results are
+    /// exactly one per input block, regardless of how many blocks needed to be
+    /// executed. To avoid surprises in how many blocks are actually executed,
+    /// `blocks_per_block` must be specified and the function will panic if it
+    /// is wrong.
+    #[track_caller]
     pub(crate) fn measure_blocks<'a>(
         &'a mut self,
         blocks: Vec<Vec<SignedTransaction>>,
+        blocks_per_block: usize,
```

Comment:
Is there a better name here? Perhaps, `block_latency`?  We can than say that `0` latency means everything is in the same block.

Reply:
Yeah, that's so much clearer. Thanks for the idea!